### ISSUE TITLE
Flutter: FMDB fail building using Xcloud when using sqflite, locally it works fine

### ISSUE THREAD

**Issue by large** [tekartik/sqflite#961]
Hi! I am not a pro flutter developer, so I am unaware if this is correct place to ask.
My project has compiled perfectly in the cloud until Xcode 14.x (I guess?), and it failed with a static link:
![image](https://user-images.githubusercontent.com/535446/227467929-57d6bcb3-c3de-4e4b-a8f1-47ad93475606.png)

After searching for issues regarding libarclite_iphoneos.a and links, it [recommeded updating](https://stackoverflow.com/questions/75574268/missing-file-libarclite-iphoneos-a-xcode-14-3) the podfile post install like this
```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)
    target.build_configurations.each do |config| #Added this line
        config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '13.0'
     end
  end
end
```

When do so the FMDB error was fixed, but it failed on firebase_app_check package, making the Xcloud unable to compile & publish.
![image](https://user-images.githubusercontent.com/535446/227469222-0049f48e-e584-4ce5-a3a8-0c92c7b6004f.png)

Using Xcode locally (Version 14.2 (14C18)) it compile, archive, verify and publish to testflight without any problems.

So is this an sqflite issue, or am I barking up the wrong tree? :)

**Comment by hukusuke1007** [comment]
Same issue. Archiving fails with Xcode14.3. This is log.

```
building file list ... rsync: link_stat "/Users/shohei/project/linkpocket/ios/../../../IntermediateBuildFilesPath/UninstalledProducts/iphoneos/FMDB.framework" failed: No such file or
    directory (2)
```

**Comment by alextekartik** [comment]
Maintaining iOS/MacOS xcode project is really a pain, sorry. I still have only have 14.2 (14.3 is only a release candidate, right?). The first think I would say would be to try the [example app](https://github.com/tekartik/sqflite/tree/master/sqflite/example) first to see if it happens too there first. Maybe flutter does not support fully xcode 14.3 yet?

**Comment by serhatuysal** [comment]
Can you help us, we're experiencing the same issue?

**Comment by alextekartik** [comment]
@serhatuysal I need a way to reproduce the issue first. Does it happen on Xcode 14.2? Can you reproduce with the [example app](https://github.com/tekartik/sqflite/tree/master/sqflite/example) or do you have a failing project you can share? Also does it happen on ARM or x86 based MacOS? And unfortunately I'm not sure I'm the best person to help here on the build issues for iOS as I'm mostly developing on linux...

**Comment by large** [comment]
@alextekartik: I was probably barking up the wrong tree here :)

Xcode local probably used cached stuff so even if I compiled it with 14.3 it did work earlier.
Now I have cleaned and updated everything to latest released and got the error locally too, that made it much easier to fix.

I found a solution in my project and this is what I did:
1. Disabled the firebase_app_check (what caused the "new unavailable"), that is not FMDB fault
2. Deleted the podfile.lock under "lib/ios"
3. Called "flutter clean" and then "flutter pub dev"
4. Compiled in Android Studio and got the error above again
5. Opened the Xcode 14.3 editor, changed to View / Navigators / Projects and selected "Pods" (blue icon)
6. In the list of pods select "FMDB", then tab "General" and "Minimum development iOS 11.0" not 8.0 as it defaults to all the time.

Tip2:
I had also the same error on the pod "ReachabilitySwift" locked to 8.0.
Changing that to 11.0 was success for me.

Tip3:
If you compile locally and gets the error, after compiling in Android Studio; try todo the same in Xcode pressing the "Play" button. Press the X with red background and it will show you which pod is messing with you.
Since we flutter developers are not native using Swift, the Xcode application is not something you use all the day 👍 

Thank you for the followup @alextekartik !
